packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -150,82 +219,127 @@ class MemoryBodyState extends State<MemoryBody> {
 */
   }
 
+  static const minIncludeTextLeftButtons = 1300;
+  static const minIncludeTextRightButtons = 1100;
+
   Widget _leftsideButtons() {
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.center,
-      children: [
-        OutlineButton(
-          key: MemoryScreen.pauseButtonKey,
-          onPressed: controller.paused ? null : _pauseLiveTimeline,
-          child: const MaterialIconLabel(
-            Icons.pause,
-            'Pause',
-            minIncludeTextWidth: 900,
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: 6),
+      // Add a semi-transparent background to the
+      // expand and collapse buttons so they don't interfere
+      // too badly with the tree content when the tree
+      // is narrow.
+      color: Theme.of(context).scaffoldBackgroundColor.withAlpha(200),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          OutlineButton(
+            key: MemoryScreen.pauseButtonKey,
+            onPressed: controller.paused ? null : _pauseLiveTimeline,
+            child: Label(
+              pauseIcon,
+              'Pause',
+              minIncludeTextWidth: 1300,
+            ),
           ),
-        ),
-        OutlineButton(
-          key: MemoryScreen.resumeButtonKey,
-          onPressed: controller.paused ? _resumeLiveTimeline : null,
-          child: const MaterialIconLabel(
-            Icons.play_arrow,
-            'Resume',
-            minIncludeTextWidth: 900,
+          OutlineButton(
+            key: MemoryScreen.resumeButtonKey,
+            onPressed: controller.paused ? _resumeLiveTimeline : null,
+            child: Label(
+              playIcon,
+              'Resume',
+              minIncludeTextWidth: 1300,
+            ),
           ),
-        ),
-      ],
+          const SizedBox(width: 16.0),
+          OutlineButton(
+              key: MemoryScreen.clearButtonKey,
+              onPressed: controller.memorySource == MemoryController.liveFeed
+                  ? _clearTimeline
+                  : null,
```

Comment:
why should we be unable to clear in live mode?

Reply:
clear makes no sense when loading an exported file (there should be a todo to change clear to delete).